IntegrationWithPharmacies/UrgentMedicineOrderApi/Service/SmptServerService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Net.Mail;
+using System.Text;
+using UrgentMedicineOrderApi.Model;
+
+namespace UrgentMedicineOrderApi.Service
+{
+    public class SmptServerService
+    {
+        public SmptServerService() { }
+        public void SendEMailNotificationForUrgentOrdee(UrgentMedicineOrder urgentMedicineOrder, string pharmacyName)
```

Comment:
Change name of method to SendEMailNotificationForUrgentOrder and apply a method name change.